burgle.lic

```diff
@@ -456,7 +456,7 @@ class Burgle
 
   #ripped out of steal.lic
   def put_item?(item)
-    case bput("put my #{item} in my #{@loot_container}", 'What were you', 'You put', "You can't do that", 'no matter how you arrange it', 'even after stuffing', 'The .* is *.* too \w+ to fit in', 'There isn\'t any more room', 'perhaps try doing that again')
+    case bput("put my #{item} in my #{@loot_container}", 'What were you', 'You put', "You can't do that", 'no matter how you arrange it', 'even after stuffing', 'The .* is *.* too \w+ to fit in', 'There isn\'t any more room', 'perhaps try doing that again', 'That\'s too heavy to go in there')
```

Comment:
This is also a good candidate to switch to `DRCI.put_away_item?(item, @loot_container)`. There's likely some other snippets in this script that could switch to newer common methods that didn't exist at the time.

Reply:
> 

Great point, especially since DRCI can hold the exhaustive list of failure patterns. (And sure enough, this failure pattern is collected there.)

This was at least a quick fix so burgle doesn't hang and risk getting folks caught.